MODULE.bazel

```diff
@@ -854,6 +854,23 @@ filegroup(
     ],
 )
 
+http_archive(
+    name = "bitcoin_core_linux_aarch64",
```

Comment:
do you know why this wasn't needed before? do we care about building those on aarch64-linux?

Reply:
because we explicitly listed the following test targets before:
```
//rs/pocket_ic_server:test
//rs/pocket_ic_server:gateway
```
but not `//rs/pocket_ic_server:bitcoin_integration_tests`.